docs/modules/ROOT/pages/api_reference.adoc

```diff
@@ -974,6 +981,43 @@ curl --location --request POST 'http://localhost:8080/api/v1/relayers/<stellar_r
 
 NOTE: WASM can be provided as either `"type": "base64"` or `"type": "hex"` encoding.
 
+**5. Submit Pre-built Unsigned Transaction (XDR):**
+
```

Comment:
Let's create relayer-sdk examples for each type

Reply:
We also need to create new openapi file and update relayer sdk to include types for new tx types that are introduced with this PR.